### DISCUSSION TITLE
use  <a/> or <nuxt-link/>

### DISCUSSION THREAD

**Discussion by lijx2018github** [nuxt-modules/i18n#1388] [Q&A] [answered]

**Comment by lijx2018github** [comment]
两个标签同时存在，<a>标签跳转都没问题
![p1](https://user-images.githubusercontent.com/45417524/152983876-750ef041-5617-43f1-a8e0-39b1c0d6954d.png)
)
![b1](https://user-images.githubusercontent.com/45417524/152983947-0ef324c9-59c0-4c42-b2c0-73ebcdb8ac03.gif)

如果只写<a>标签则无法成功跳转
![p2](https://user-images.githubusercontent.com/45417524/152984079-10535523-2113-4ae5-9987-ec01a44d3d80.png)
![b2](https://user-images.githubusercontent.com/45417524/152984090-c7cf0b4f-a87d-40d5-9588-8661e5c3f481.gif)

**Comment by lijx2018github** [comment]
```
nuxt.config.js

modules: [
    '@nuxtjs/i18n',
    '@nuxtjs/axios',
    '@nuxtjs/style-resources'
  ],
  i18n: {
    locales: [
      { code: "lang_tw", iso: "zh-HK", file: "tw.json" },
      { code: "lang_en", iso: "en-US", file: "en.json" }
    ],
    langDir: "~/locales/", 
    lazy: true,
    strategy: "prefix_except_default",
    defaultLocale: "lang_tw",
    vueI18n: {
      fallbackLocale: ['lang_tw'],
    },
    sortRoutes: true,
    detectBrowserLanguage: { 
      useCookie: true,
      cookieKey: "lang",
      redirectOn: 'root',
      alwaysRedirect: true,
      fallbackLocale: "lang_tw"
    }, 
  }
```

**Comment by lijx2018github** [comment]
help me please

**Comment by rchl** [comment]
Use `@click.prevent`. Otherwise the browser will navigate to the link of the anchor.

**Comment by lijx2018github** [comment]
@rchl  thanks, does this affect the SEO of ` / about '

**Comment by rchl** [comment, marked as answer]
For better SEO use
```
<a :href="localePath('/global/about')" @click.prevent="$router.push(pageRoute)">...</a>
```

Or just the `nuxt-link`.

**Comment by lijx2018github** [comment]
@rchl   thanks so much
